src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,18 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null,
+}
```

Comment:
```suggestion
interface Props {
  user: User,
}
```
Don't render the `UserInfo` component if no user is present